lighthouse-cli/test/smokehouse/test-definitions/perf/expectations.js

```diff
@@ -240,11 +240,8 @@ module.exports = [
           animations: [
             {
               name: 'anim',
-              // The animation reliably gets kUnsupportedCSSProperty (1 << 13) === 8192.
```

Comment:
Is this comment no longer relevant when after the switch?

Reply:
Yeah, it was only relevant for `background-color`